### ISSUE TITLE
How many params does nougat have?

### ISSUE THREAD

**Issue by brando90** [facebookresearch/nougat#156]
Based on the README, Nougat has two main models:

- Nougat small: Around 250 million parameters
- Nougat base: Around 350 million parameters

The README mentions that the base model uses a Swin Transformer encoder and an mBART decoder. The mBART decoder has 10 layers and the whole architecture has 350M parameters. 

The smaller model has a slightly smaller sequence length and only 4 decoder layers, with a total of 250M parameters.

So in summary:

- Nougat small: 250 million parameters
- Nougat base: 350 million parameters

Am I right?

**Comment by lukas-blecher** [comment]
Yes, correct.

**Comment by xsank** [comment]
Will the nougat provide large model later?
